### DISCUSSION TITLE
Tenderbake / Ithacanet support in Singatory

### DISCUSSION THREAD

**Discussion by jevonearth** [ecadlabs/signatory#157] [General]
We are currently running (dog-fooding) a pre-release version of Signatory on the Ithaca testnet. Signatory is doing all the signing for testnet baker: `tz1RuHDSj9P7mNNhfKxsyLGRDahTX5QD1DdP`

We will cut a new relaease of Signatory with full Tenderbake support in the coming week!

**Comment by JarrodWoodard** [comment]
If someone will walk me through how to set it up, I will help out. :)
